source/testing/testing-controllers.md

```diff
@@ -1,8 +1,9 @@
-_Unit testing methods and computed properties follows previous patterns shown
-in [Unit Testing Basics] because Ember.Controller extends Ember.Object._
+_Container testing methods and computed properties follows previous patterns shown
+in [Testing Basics] because Ember.Controller extends Ember.Object._
 
-Unit testing controllers is very simple using the unit test helper which is part
-of the ember-qunit framework.
+Testing controllers is very simple using the `setupTest` helper which is part
```

Comment:
describing something as "simple" in documentation can be problematic for people that are learning about this stuff and don't consider it simple yet.